CoreFeatures/Base/Pool/src/main/java/info/smart_tools/smartactors/base/pool/Pool.java

```diff
@@ -110,4 +127,11 @@ public void onAvailable(final IActionNoArgs action) {
             throw new RuntimeException("Failed to execute ActionNoArgs", e);
         }
     }
+
+    private boolean validateItem(Object item) {
+        if (validationFunction == null) {
+            return true;
+        }
```

Comment:
:monocle_face: It's possible to avoid this `if` if use the default validation function like `(item) -> true`.

Reply:
yes, should be much better, will change today

Reply:
Updated